lib/error.ex

```diff
@@ -8,7 +8,6 @@ defimpl AshGraphql.Error, for: Ash.Error.Changes.InvalidChanges do
       message: error.message,
       short_message: error.message,
       vars: Map.new(error.vars),
-      code: Ash.ErrorKind.code(error),
```

Comment:
This probably will break existing code which expects a `code` inside error, is there anything else we can use insead of `Ash.ErrorKind.code`?

Reply:
Yeah, so unfortunately what has to be done here for backwards compatibility is for the original error codes to be hand-copied here. Basically responsibility for generating this additional metadata has been moved out of core and into the api extensions. So this one would be something like `"invalid_changes"`, and so-on.